_posts/fr/newsletters/2022-09-07-newsletter.md

```diff
@@ -0,0 +1,65 @@
+---
+title: 'Bitcoin Optech Newsletter #216'
+permalink: /fr/newsletters/2022/09/07/
+name: 2022-09-07-newsletter
+slug: 2022-09-07-newsletter
```

Comment:
I think you need to add a '-fr' suffix to the _slug_ and _name_ properties or the make file will throw errors.

Reply:
Done, 
but `make production` is still in error :
bundle exec _contrib/schema-validator.rb _data/schemas/compatibility.yaml _data/compatibility/blockchaininfo.yaml
/usr/bin/env: ‘ruby\r’: No such file or directory
make: *** [Makefile:120: _site/en/compatibility/blockchaininfo/index.html] Error 127

Reply:
`bundle exec jekyll serve` is working fine

Reply:
Looks like the issue is trailing spaces. Make sure to remove all of them.
I was able to build your version after fixing the spaces and running _make clean_ and then _make_

Reply:
I have an error : 
cyrille@PC-Commun:/mnt/c/Users/cyril/OneDrive/Documents/GitHub/bitcoinops.github.io$ make
bundle exec _contrib/schema-validator.rb _data/schemas/compatibility.yaml _data/compatibility/blockchaininfo.yaml
/usr/bin/env: ‘ruby\r’: No such file or directory
make: *** [Makefile:120: _site/en/compatibility/blockchaininfo/index.html] Error 127

Reply:
But it's work well with this command : ` bundle exec jekyll serve`